components/legacy_sidebar/header/dropdown/sidebar_header_dropdown.jsx

```diff
@@ -74,6 +78,8 @@ export default class SidebarHeaderDropdown extends React.PureComponent {
                     teamDisplayName={this.props.teamDisplayName}
                     teamId={this.props.teamId}
                     openModal={this.props.actions.openModal}
+                    getFirstAdminVisitMarketplaceStatus={this.props.actions.getFirstAdminVisitMarketplaceStatus}
+                    showUnread={isAdmin(this.props.currentUser.roles) && !this.props.firstAdminVisitMarketplaceStatus}
```

Comment:
Perhaps the SidebarHeaderDropdownButton could get the action and derive showUnread itself since the SidebarHeaderDropdown doesn't seem to care about these other than to just pass them along

Reply:
That was the original change - then after addressing this comment: https://github.com/mattermost/mattermost-webapp/pull/7563#discussion_r581923842 - it was moved up the stack - I'm 0/5 on this - I can see both sides.